ledger/notary-plugin/src/main/java/net/corda/v5/ledger/notary/pluggable/package-info.java

```diff
@@ -0,0 +1,4 @@
+@Export
+package net.corda.v5.ledger.notary.pluggable;
```

Comment:
All these interfaces in this package are needed to *create a plugin*, not related to an actual plugin. I'd argue it's better to be in `pluggable` since they are required to create a plugin thus "pluggable". 

A `plugin-core` module name would make sense to me.

That's just my opinion though, if you still want to stick with `plugin` I can modify it.

Reply:
Discussed this offline, will stick to having `plugin` as the root package and `api`, `core` as sub-packages.